src/libraries/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs

```diff
@@ -71,6 +80,60 @@ public void SettingUpdatesProperties()
             });
         }
 
+        [Fact]
+        public void SettingUpdatesProperties()
+        {
+            T item = GetExistingItem();
+            SettingUpdatesPropertiesCore(item);
+        }
+
+        [Theory]
+        [PlatformSpecific(~TestPlatforms.Browser)] // Browser is excluded as it doesn't support symlinks
+        [InlineData(false, false)]
+        [InlineData(false, true)]
+        [InlineData(true, false)]
+        [InlineData(true, true)]
+        public void SettingUpdatesPropertiesOnSymlink(bool targetIsRelative, bool targetExists)
```

Comment:
I thought it might be a good test case and @Jozkee agreed, it is probably unnecessary though. https://github.com/dotnet/runtime/pull/52639#discussion_r736013152

Reply:
@Jozkee the additional argument checks whether both relative and full paths work as the symlink target path. It's not related to the functionality that gets tested here.

Reply:
The reason that we included it here is we're checking if it changes the date on the target file as well, so all variants of the 'target' should be included IMO (which is what it currently is: relative/absolute, exists/doesn't, file/folder, info/static).

Reply:
> all variants of the 'target' should be included IMO

Another variant would be using multiple links to get to the target.

Because the target gets resolved by the OS, I don't see the need use different target paths.

I'm ok if you want to keep it.

Reply:
> Another variant would be using multiple links to get to the target.

That's true. I don't see it as necessary personally.

> Because the target gets resolved by the OS, I don't see the need use different target paths.

Probably fair. I'd rather it be there, if @Jozkee wants I can remove it.

> I'm ok if you want to keep it.

👍 thanks

Reply:
What @tmds says makes sense. You can remove `targetIsRelative` if you want.